python/caffe/coord_map.py

```diff
@@ -76,6 +76,7 @@ def coord_map(fn):
         axis -= 1  # -1 for last non-coordinate dim.
         return axis, 1, - offset
     else:
+        print("Error: Encountered an unfamiliar layer: " + fn.type_name)
```

Comment:
Why not instead put this string in the exception as a message?  It's unconventional to log an error to stdout and then raise a separate exception.